esp/esp/program/tests.py

```diff
@@ -121,9 +175,12 @@ def testUserSearchFn(self):
         self.assertStringContains(response['location'], "/manage/userview?username=testuser123543")
 
         # Full name, unique
-        response = c.get("/manage/usersearch", { "userstr": "Admin User" })
```

Comment:
I don't think we should take out this test case; otherwise, the tests never hit the code path for full name searches.